sqlglot/dialects/tsql.py

```diff
@@ -242,6 +243,7 @@ class Tokenizer(tokens.Tokenizer):
 
         KEYWORDS = {
             **tokens.Tokenizer.KEYWORDS,
+            "AS OF": TokenType.ASOF,
```

Comment:
asof is a common keyword, i think we can put this in the base?

Reply:
Should we just do "AS OF" or "ASOF" too? (the latter is used in clickhouse).

Reply:
only as of, asof is not safe

Reply:
hm maybe not since you could do "select 1 as of"

Reply:
yea forget it

Reply:
Lol right, missed this too. Reverting the last commit then

Reply:
it's probably unsafe to do this in tsql too then, I'll just `match_text_seq`